src/main/java/org/apache/commons/beanutils2/converters/PatternConverter.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.commons.beanutils2.converters;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+/**
+ * Converts a regular expression into a Java {@link Pattern} object.
+ *
+ * @since 2.0.0
+ */
+public class PatternConverter extends AbstractConverter {
+
+    /**
+     * Construct a <b>{@link Pattern}</b> <i>Converter</i> that throws
+     * a {@code ConversionException} if an error occurs.
+     */
+    public PatternConverter() {
+        super();
+    }
+
+    /**
+     * Constructs a {@link org.apache.commons.beanutils2.Converter} that will return
+     * the specified default value if a conversion error occurs.
+     *
+     * @param defaultValue The default value to be returned
```

Comment:
For all files, the checktyle settings in this components allows for line length to 120, so use them :-)